docker-compose.yaml

```diff
@@ -16,30 +16,43 @@ version: '3.5'
 
 services:
   tcf:
```

Comment:
change the service name to "avalon-shell"

Reply:
Ok